lite/types.go

```diff
@@ -0,0 +1,13 @@
+package lite
+
+import (
+	"github.com/tendermint/tendermint/types"
+)
+
+// Certifier checks the votes to make sure the block really is signed properly.
+// Certifier must know the current or recent set of validitors by some other
+// means.
+type Certifier interface {
```

Comment:
Certifier defined it types.go and Provider in provider.go. I am not sure that Certifier is the best name. Maybe it's more Verifier than Certifier.

Reply:
I agree.

Reply:
TODO make new issue